scripts/zones/Crawlers_Nest_[S]/mobs/Morille_Mortelle.lua

```diff
@@ -12,6 +14,16 @@ function onMobDeath(mob, player, isKiller)
 end;
 
 -----------------------------------
+-- onMobSpawn
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:setMod(MOD_DOUBLE_ATTACK, 20);
+	mob:setMod(MOD_STORETP, 10);
```

Comment:
sorry i was being helped by someelse copy and paste from email. They saw my code and told me it was missing these items which I didnt even know had to be in the Lua. Every mistake I learn from though.
i went to edit this file but must have been changed i counted 4 spaces or 8 on all lines